odoo/osv/query.py

```diff
@@ -130,6 +130,22 @@ def select(self, *args):
         )
         return query_str, params
 
+    def subselect(self, *args):
+        """ Similar to :meth:`select`, but for sub-queries.
```

Comment:
```suggestion
        """ Similar to :meth:`.select`, but for sub-queries.
```
Probably safer / more reliable: `:meth:` will start looking up case-insensitively from the top of the module so if we ever add a `select` free function or a `Select` object / class those will be matched before the method.

With the `.` prefix, it will perform the look up starting from the class instead, which shoudl always find the method.